wordpress-mobile/WordPress-FluxC-Android
# [Coupons] Temporarily remove Products and ProductCategories Room tables

This PR removes the Products and ProductCategories Room tables, which are not needed and will be readded later, when we eventually migrate to Room for all the functionality.

**To test:**
1. Make sure the coupons can be fetched and updated 
2. Make sure the unit tests pass
